rllib/train.py

```diff
@@ -430,8 +430,8 @@ def run_rllib_experiments(
 
     checkpoints = []
     for trial in trials:
-        if trial.checkpoint.dir_or_data:
-            checkpoints.append(trial.checkpoint.dir_or_data)
+        if trial.checkpoint.path:
+            checkpoints.append(trial.checkpoint.path)
```

Comment:
```suggestion
        if trial.checkpoint:
            checkpoints.append(trial.checkpoint)
```

Reply:
all we use this for is a print later on. nicer to include the fs in the string

Reply:
Also not sure if this is necessary, but this suggestion would technically allow this to be backwards compatible (though I'm not sure what the output representation ends up looking like)

Reply:
Updated